specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -2047,6 +2047,41 @@
         }
       }
     },
+    "/providers/Microsoft.Management/managementGroups/{groupId}/providers/{resourceProviderNamespace}/register": {
```

Comment:
Stable swagger is NOT supposed to be modified. Do you know if SDKs have been released on this api-version?

Reply:
My understanding is that this is a swagger bug fix rather than adding a new API. @tjegbejimba can you confirm that this is the case?

Reply:
The MG registration capability is already on the service side it just hasn't been added to the swagger yet. We're not adding a new API